builtins/safe-default-configuration.txt

```diff
@@ -169,3 +169,65 @@ th
 - lang
 - title
 
+// MathML, per https://github.com/WICG/sanitizer-api/issues/103#issuecomment-2591328351
```

Comment:
Would it make sense to add a link to https://w3c.github.io/mathml-core/#mathml-elements-and-attributes
and listed elements that are purposely omitted (I understand it's mphantom, maction, annotation and annotation-xml)? Similar to what we do with "meta and link, purposely omitted".

Reply:
Done.